src/commands/generateEditorConfig.ts

```diff
@@ -1,16 +1,27 @@
-import {exists, writeFile} from 'fs';
+import {
+	exists,
+	writeFile
+} from 'fs';
 import * as path from 'path';
-import {workspace, window} from 'vscode';
-import {Utils} from '../utils';
+import {
+	workspace,
+	window
+} from 'vscode';
+import {
+	Utils
+} from '../utils';
```

Comment:
We could use `export default class Utils` and then use `import Utils from '../utils';`.